### PR TITLE
Formatting and Typing Enhancements

### PR DESCRIPTION
**Pull request by wintron04**

### CODE REVIEW

File: src/berry_mill/kiwrap.py [hunk 1]
```diff
@@ -17,37 +17,39 @@
...
         if self._kiwiparams.get("debug"):
             self._kiwi_options.append("--debug")
-            
+
         log.info("Using appliance \"{}\" located at \"{}\"".format(self._appliance_descr, self._appliance_path))
         try:
             config_tree = etree.parse(f"{self._appliance_descr}")
         except Exception as err:
-            log.warning("Failure while parsing appliance description", exc_info= err)
+            log.warning("Failure while parsing appliance description", exc_info=err)
```
Comment (osab270844) [inline src/berry_mill/kiwrap.py:45]: Actually that should be log.error.

File: src/berry_mill/kiwrap.py [hunk 2]
```diff
@@ -17,37 +17,39 @@
...
             self.cleanup()
             sys.exit(1)
 
         try:
             profiles = config_tree.xpath("//profile/@name")
         except Exception as err:
-            log.warning("Failure while trying to extract profile names", exc_info= err)
+            log.warning("Failure while trying to extract profile names", exc_info=err)
```
Comment (osab270844) [inline src/berry_mill/kiwrap.py:52]: same here.

File: src/berry_mill/kiwrap.py [hunk 3]
```diff
@@ -17,37 +17,39 @@
 
 log = kiwi.logging.getLogger('kiwi')
 
+
```
Comment (osab270844) [inline src/berry_mill/kiwrap.py:20]: No need for extra empty line.

File: src/berry_mill/kiwiapp.py [hunk 4]
```diff
@@ -1,35 +1,38 @@
...
+import pathlib
 from typing import Dict, List
 from abc import ABC, abstractmethod
 
 from berry_mill.boxbuild import BoxBuildTask
 from berry_mill.localwrap import LocalBuildTask
-import pathlib
 from berry_mill.preparetask import PrepareTask
 
+
```
Comment (osab270844) [inline src/berry_mill/kiwiapp.py:14]: same.

File: src/berry_mill/kiwiapp.py [hunk 5]
```diff
@@ -79,40 +79,40 @@ def _get_relative_path(self) -> str:
         containing the repo specific kiwi arguments,
         like --add-repo and --ignore-repos.
 
-        The path will be relative the the boxroot in the 
+        The path will be relative the the boxroot in the
```
Comment (osab270844) [inline src/berry_mill/kiwiapp.py:82]: "the the", you mean "to the"?

File: src/berry_mill/params.py [hunk 6]
```diff
@@ -13,6 +13,7 @@ class KiwiParams(TypedDict):
     profile: str
     debug: bool
 
+
```
Comment (osab270844) [inline src/berry_mill/params.py:16]: same

File: src/berry_mill/params.py [hunk 7]
```diff
@@ -34,6 +35,7 @@ class KiwiBuildParams(KiwiParams):
     target_dir: str
     no_accel: bool
 
+
```
Comment (osab270844) [inline src/berry_mill/params.py:38]: same

File: src/berry_mill/preparer.py [hunk 8]
```diff
@@ -1,33 +1,35 @@
...
+from kiwi.exceptions import KiwiPrivilegesError, KiwiRootDirExists
 from .kiwiapp import KiwiAppPrepare
 from .kiwrap import KiwiParent
 from .params import KiwiPrepParams
-import kiwi.logger
-from kiwi.exceptions import KiwiPrivilegesError, KiwiRootDirExists
 
 log = kiwi.logging.getLogger('kiwi')
 
+
```
Comment (osab270844) [inline src/berry_mill/preparer.py:11]: same

File: src/berry_mill/preparetask.py [hunk 9]
```diff
@@ -4,28 +4,29 @@
 
 log = kiwi.logging.getLogger('kiwi')
 
+
```
Comment (osab270844) [inline src/berry_mill/preparetask.py:7]: extra line.

**Review by isbm — APPROVED** [review]
@osab270844 these are actually correct, as this is all coming from PEP8 autoformatter. In fact, we want to auto-format everything and then later if this applies to every new PR.

**Pull request merged**